WordPress/Classes/Utility/WPTableViewHandler.m

```diff
@@ -205,12 +205,12 @@ - (void)refreshTableViewPreservingOffset
 
 - (void)cacheRowHeight:(CGFloat)height forIndexPath:(NSIndexPath *)indexPath
 {
-    [self.cachedRowHeights setObject:@(height) forKey:[indexPath toString]];
+    [self.cachedRowHeights setObject:@(height) forKey:indexPath];
```

Comment:
I think this could be a problem. I remember debugging this, and it was the reason why `toString` was added as an extension: TableView may provide different instances of `NSIndexPath`, for the same coordinates, when you're scrolling.

And since indexPath isn't "Hashable", the cache might not be picked up. Can we revert this change?